Libraries/Text/RCTTextViewManager.m

```diff
@@ -26,6 +26,13 @@ - (UIView *)view
 
 RCT_REMAP_VIEW_PROPERTY(autoCapitalize, textView.autocapitalizationType, UITextAutocapitalizationType)
 RCT_EXPORT_VIEW_PROPERTY(autoCorrect, BOOL)
+RCT_CUSTOM_VIEW_PROPERTY(spellCheck, NSString, RCTTextView)
```

Comment:
Instead of repeating the logic here, extract this logic to an RCTConvert category.